internals/src/Tizen.Peripheral/Tizen.Peripheral/Gpio.cs

```diff
@@ -168,7 +168,16 @@ private void SetIntteruptedCallback()
 
         private void OnInterrupted(IntPtr handle, ErrorCode error, IntPtr data)
         {
-            ValueChanged?.Invoke(this, new PinUpdatedEventArgs(PinNumber, Read()));
+            NativeGpio.PeripherialGpio pio = (NativeGpio.PeripherialGpio)
+                                            System.Runtime.InteropServices.Marshal.PtrToStructure(handle,
+                                                typeof(NativeGpio.PeripherialGpio));
+            // check magic values to verify capi structures integrity
+            if (pio.vermagic != 13712 || pio.cbInfo.vermagic != 14469)
+            {
+                throw ExceptionFactory.CreateException(ErrorCode.BadAddress);
```

Comment:
Do not throw exception in Callback, nobody to catch it

Reply:
Hm.. I don't know which is better, throw excpetion and terminate app, or just print out log and ignore this callcak.

And I don't know why do you need magic value check, because TizenFX is including on platform binary and it always compatiable with library built in platform binary

Reply:
And it is not big issue, so i approve it

Reply:
As you mentioned callback should not throw exception, so I will change `throw` exception to `TIzen.Log`.
I want to check structure magic number to check if it's parsed correctly rather than using it as it -- read arbitrary data pointed by handle.